src/components/EventForm.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+import { DateRangePicker } from "react-dates";
+import moment from "moment";
+import uuid from "uuid";
+
+export default class EventForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: "",
+      description: "",
+      organisator: "",
+      city: "",
+      category: "",
+      image: false,
+      id: uuid(),
+      createdAt: moment(),
+      startDate: moment(),
+      startDateId: "",
+      endDate: moment(),
+      endDateId: "",
+      calendarFocused: null,
+    };
+  }
+
+  onTitleChange = e => {
+    const title = e.target.value;
+    this.setState(() => ({ title }));
+  };
+
+  onDescriptionChange = e => {
+    const description = e.target.value;
+    this.setState(() => ({ description }));
+  };
+
+  onOrganisatorChange = e => {
+    const organisator = e.target.value;
+    this.setState(() => ({ organisator }));
+  };
+
+  onCityChange = e => {
+    const city = e.target.value;
+    this.setState(() => ({ city }));
+  };
+
+  onCategoryChange = e => {
+    const category = e.target.value;
+    this.setState(() => ({ category }));
+  };
+
+  onImageChange = e => {
+    const image = e.target.value;
+    this.setState(() => ({ image }));
+  };
+
+  onFocusChange = calendarFocused => {
+    this.setState(() => ({ calendarFocused }));
+  };
+
+  onDatesChange = ({ startDate, endDate }) => {
+    this.setState(() => ({ startDate, endDate }));
+  };
+
+  onSubmit = e => {
+    e.preventDefault();
+    this.props.onSubmit({
+      title: this.state.title,
+      description: this.state.description,
+      organisator: this.state.organisator,
+      city: this.state.city,
+      category: this.state.category,
+      startDate: this.state.startDate.valueOf(),
+      endDate: this.state.endDate.valueOf(),
+      image: this.state.image,
+      createdAt: this.state.createdAt.valueOf(),
+      id: this.state.id,
+    });
+  };
+
+  render() {
+    return (
+      <div>
```

Comment:
You can use `<>...</>` instead of `<div>...</div>` :)

Reply:
Also, if you have `form` as the only direct child then you don't need neither `div` nor `<>`